react/features/subtitles/components/TranscriptionSubtitles.native.js

```diff
@@ -0,0 +1,65 @@
+// @flow
+
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { Container, Text } from '../../base/react';
+
+import {
+    _abstractMapStateToProps,
+    AbstractSubtitles,
+    type AbstractSubtitlesProps
+} from './AbstractSubtitles';
+import styles from './styles';
+
+type Props = AbstractSubtitlesProps & {
+    onPress: Function
+};
+
+/**
+ * React {@code Component} which can display speech-to-text results from
+ * Jigasi as subtitles.
+ */
+class TranscriptionSubtitles
+    extends AbstractSubtitles<Props> {
+
+    /**
+     * Renders the transcription text.
+     *
+     * @param {string} id - The ID of the transcript message from which the
+     * {@code text} has been created.
+     * @param {string} text - Subtitles text formatted with the participant's
+     * name.
+     * @returns {React$Element} - The React element which displays the text.
+     * @protected
+     */
+    _renderParagraph(id: string, text: string): React$Element<*> {
+        return (
+            <Text
+                key = { id }
+                onPress = { this.props.onPress }
```

Comment:
I see what you want to do here... but do you think it's worth it? How high can the text get? If it will obstruct the view a lot, then yeah, we need it.

Reply:
From my testing it was pretty often when the text was blocking the clicks that's why I've added it.